spec/bucketing_spec.rb

```diff
@@ -75,9 +75,9 @@ def get_bucketing_id(user_id, entity_id=nil)
     expect(bucketer).to receive(:generate_bucket_value).once.and_return(3000)
 
     experiment = config.get_experiment_from_key('group1_exp2')
-    expect(bucketer.bucket(experiment, 'test_user')).to be_nil
+    expect(bucketer.bucket(experiment,'bucket_id_ignored', 'test_user')).to be_nil
```

Comment:
what does the bucketing key: `bucketing_id_ignored` do?
how is it ignored?

Reply:
I have used this string in all those cases where we mock  the method generate_bucket_value to test traffic allocations as in above
expect(bucketer).to receive(:generate_bucket_value).twice.and_return(3000)
This makes it more readable to the developer that the string passed as bucketing id does not have an impact